app/controllers/api/invitations_controller.rb

```diff
@@ -10,6 +10,18 @@ def create
     respond_with_collection
   end
 
+  def pending
+    load_and_authorize :group
```

Comment:
Need to check that the user is a group admin. IE can :view_pending_invitations, @group